contracts/DomainRegistry.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "solidity-stringutils/src/strings.sol";
+import {Ownable} from "./Ownable.sol";
+
+/// @author Serhii Smirnov
+/// @title Describes a registry of all registered domain names linked to a domain holder, who have registered the domain name
+contract DomainRegistry is Ownable {
+    using strings for *;
+
+    /// @notice Price for registration sub-domains 
+    uint public registrationPrice;
+
+    /// @notice Mapping of top-level domain name to domain holder address
+    mapping(string => address payable) public domainsMap;
+
+    
+    /// @notice Event that is notifying external world about new domain registration action
```

Comment:
// NatSpec comments syntax is not correct
// for all multiline comments you need to use /** */ instead of ///